src/wasm-lib/kcl/tests/artifact_graph_example_code1/ops.snap

```diff
@@ -78,13 +109,76 @@ snapshot_kind: text
   {
     "labeledArgs": {
       "data": {
+        "artifactIds": [
+          "[uuid]"
+        ],
         "sourceRange": [
           328,
           338,
           0
         ]
       },
       "tag": {
+        "value": {
+          "type": "TagIdentifier",
+          "type": "TagIdentifier",
+          "value": "seg02",
+          "info": {
+            "type": "TagEngineInfo",
+            "id": "[uuid]",
+            "sketch": "[uuid]",
+            "path": {
+              "__geoMeta": {
+                "id": "[uuid]",
+                "sourceRange": [
+                  125,
+                  150,
+                  0
+                ]
+              },
+              "from": [
+                5.550000000000001,
+                5.0
+              ],
+              "tag": {
+                "end": 149,
+                "start": 143,
+                "type": "TagDeclarator",
+                "value": "seg02"
+              },
+              "to": [
+                5.550000000000001,
+                -5.0
+              ],
+              "type": "ToPoint"
+            },
+            "surface": {
+              "faceId": "[uuid]",
+              "id": "[uuid]",
+              "sourceRange": [
+                125,
+                150,
+                0
+              ],
+              "tag": {
+                "end": 149,
+                "start": 143,
+                "type": "TagDeclarator",
+                "value": "seg02"
+              },
+              "type": "extrudePlane"
+            }
+          },
```

Comment:
It looks like tags are still getting their full `KclValue` here

Reply:
If we don't need them, I'll happily remove them.

Reply:
I removed tag declarators and tag identifiers.